<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bitmap Font Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>500</width>
         <height>500</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="panelWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>242</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>242</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_width">
           <property name="text">
            <string>width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edWidth">
           <property name="text">
            <string>16</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_height">
           <property name="text">
            <string>height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edHeight">
           <property name="text">
            <string>16</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="chkDrawLines">
         <property name="text">
          <string>Draw lines</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkAntiAlasing">
         <property name="text">
          <string>Antialiasing</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFont">
         <property name="text">
          <string>Select Font</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBackGroundColor">
         <property name="text">
          <string>Background Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSaveAs">
         <property name="text">
          <string>Save as</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
